include/CMakeLists.txt

```diff
@@ -11,7 +13,8 @@ add_library(
 )
 
 target_include_directories(
-    atomic_queue
+    ${PROJECT_NAME}
     INTERFACE
-    ${CMAKE_CURRENT_SOURCE_DIR}
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+    $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
```

Comment:
Okay, but we don't have `install(...)` set up for this project. If we want to do this then we need to provide the installation config as well.